README.md

```diff
@@ -56,7 +56,15 @@ The simplest use case would be:
   ```
 
 `beaver docker build` is just a thin wrapper over `docker build` to add some
-default options (`--pull -t beaver` in particular).
+default options (`--pull -t $img` in particular).
+
+The image name will be taken from the `BEAVER_DOCKER_IMG` environment variable,
+if present. If not present it falls back to the result of `git config
+hub.upstream` (in case you are using the
```

Comment:
In that case I guess `TRAVIS_REPO_SLUG` would have to have a higher priority than `origin`, since both would exist in travis env.